cypress/integration/entitybrowser_spec.js

```diff
@@ -11,62 +11,80 @@ describe('Entity Browser', function() {
 
   describe('ListView', () => {
     it('should load and display essential parts of the list view', function() {
+      cy.contains('Daten werden geladen...', {timeout: 15000})
       cy.get('[title="Vorname"]')
       cy.get('[type="submit"]')
+      cy.contains('Einträgen')
       cy.contains('Aktionen auf Alle')
-      cy.get('[data-list-cell]').should('have.length.above', 1)
+      cy.get('[data-cy=list-cell]').then(el => {
+        cy.wrap(el).should('have.length.above', 10)
+      })
     })
   })
 
   describe('DetailView', () => {
     beforeEach(() => {
-      cy.get('[data-list-cell]').first()
-        .click()
+      cy.get('[data-cy=list-cell]').then(el => {
+        const firstCell = el.first()
+        cy.wrap(firstCell).click()
+      })
     })
 
     it('should display detail view', function() {
-      cy.get('[data-cy=cypress-form-field]')
-        .should('have.length.above', 1)
+      cy.get('[data-cy=cypress-form-field]').then(el => {
+        cy.wrap(el).should('have.length.above', 1)
+      })
     })
 
     it('should change value in detail view and save', function() {
-      cy.get('input#input-detailForm-callname')
-        .type('{selectall}{del}Test Callname')
-        .should('have.value', 'Test Callname')
+      cy.get('input#input-detailForm-callname').then(el => {
+        cy.wrap(el)
+          .type('{selectall}{del}Test Callname')
+          .should('have.value', 'Test Callname')
+      })
       cy.get('[data-cy=detail-form_submit-button]').children().first()
         .click()
-      cy.contains('Zuletzt gespeichert')
+      cy.contains('Zuletzt gespeichert', {timeout: 4000})
     })
 
     it('should change page back to list view without warning', function() {
-      cy.get('[data-cy=entity-detail_back-button]').children().first().children().first()
-        .click()
-      cy.get('[data-list-cell]').should('have.length.above', 1)
+      cy.get('[data-cy=entity-detail_back-button]').children().first().then(el => {
+        cy.wrap(el).click()
+      })
+      cy.get('[data-cy=list-cell]').then(el => {
```

Comment:
would it be better to make a selector on the list view rather than check if after the click on "BACK" you see some records?